src/Application.php

```diff
@@ -60,10 +61,12 @@ class Application extends SymfonyApplication
 
     private Container $container;
     private readonly EnvFlags $envFlags;
+    private readonly CacheDirectoryFactory $cacheDirectoryFactory;
 
     public function __construct(string $appName, string $appVersion)
     {
         parent::__construct($appName, $appVersion);
+        $this->cacheDirectoryFactory = new CacheDirectoryFactory();
```

Comment:
It's a small thing, but could you group assignments and left blank line after `parent` call?
Like:
```php
parent::__construct($appName, $appVersion);

$this->cacheDirectoryFactory = new CacheDirectoryFactory();
$this->envFlags = new EnvFlags();
```